### ISSUE TITLE
Can not reproduce

### ISSUE THREAD

**Issue by baterflyrity** [ZhuangShuoH/geospatial-object-detection#2]
Hi!

Im very interested in building this project for my own computer vision system and also ive read your paper. 

**How can I reproduce this network?** 

Actually ive tested several things:
1) I have a lot of missing errors so taken them from [parent repo](https://github.com/experiencor/keras-yolo3).
2) Also used latest tensorflow docker container with nvidia gpus.
3) Unfortunately always cpus are used.
4) Another pitty is that training fails with error "loss didnt improve from 8 epoch"

So can you please help to run this network? _It would be great if someone published working docker container derived from nvidia/cuda one._

**Comment by Dr-Zhuang** [comment]
The reason that you can not reproduce the network is the configuration issue. You may follow the instructions of installing and training from [experiencor-keras-yolo3](https://github.com/experiencor/keras-yolo3). 

In my experiments, the loss droped to less than 10 and kept stable.

**Comment by Dr-Zhuang** [comment]
The reason that you can not reproduce the network is the configuration issue. You may follow the instructions of installing and training from [experiencor-keras-yolo3](https://github.com/experiencor/keras-yolo3). 

In my experiments, the loss droped to less than 10 and kept stable.

**Comment by baterflyrity** [comment]
> I have a lot of missing errors so taken them from parent repo.

As ive mentioned above this instructions dont work((

Can u upload trained weights pls?

**Comment by Dr-Zhuang** [comment]
The pre-trained model was here [baiduyun](https://pan.baidu.com/s/13EmOWTGcCksQx371hIIE8g), the entry code was "7std".

**Comment by Dr-Zhuang** [comment]
The pre-trained model was here [baiduyun](https://pan.baidu.com/s/13EmOWTGcCksQx371hIIE8g), the entry code was "7std".

**Comment by baterflyrity** [comment]
> The pre-trained model was here [baiduyun](https://pan.baidu.com/s/13EmOWTGcCksQx371hIIE8g), the entry code was "7std".

Thanks but i can not download without locale phone number. Can you upload weights to google drive?

**Comment by Dr-Zhuang** [comment]
The pre-trained model was here [google drive](https://drive.google.com/file/d/1grorR7w2XntembuAm-7P0XF-qafzNxD0/view?usp=sharing) .

**Comment by baterflyrity** [comment]
Thanks. Seems to be working.
